packages/forms-web-app/src/dynamic-forms/options-question.js

```diff
@@ -93,18 +93,20 @@ class OptionsQuestion extends Question {
 				optionData.checked = (',' + answer + ',').includes(',' + optionData.value + ',');
 			}
 
-			// also need to handle dependent fields & set their answers
-			// if (optionData.conditional !== undefined) {
-			// 	let conditionalField = { ...optionData.conditional };
-			// 	conditionalField.fieldName = this.fieldName + '_' + conditionalField.fieldName;
-			// 	conditionalField.value = journey.response.answers[conditionalField.fieldName] || '';
-			// 	optionData.conditional = {
-			// 		html: nunjucks.render(
-			// 			`../views/questions/conditional/${conditionalField.type}.njk`,
-			// 			conditionalField
-			// 		)
-			// 	};
-			// }
+			// handle conditional (dependant) fields & set their answers
+			if (optionData.conditional !== undefined) {
+				let conditionalField = { ...optionData.conditional };
+
+				conditionalField.fieldName = this.fieldName + '_value';
```

Comment:
I don't know whether we have any examples of this, but it's possible to have conditional fields on more than one answer (e.g. https://design-system.service.gov.uk/components/checkboxes/#conditionally-revealing-a-related-question). By naming the conditional field `this.fieldName + '_value'` we'd potentially end up with multiple fields with the same name which would make it difficult to work out which answer goes with each selected option.

Reply:
should we use a conditional fieldname value (or use the existing value) as in the POC so you get something like:

`new-planning-conditions_yes-value`

(fieldName + conditional value + `-value`)

the -value string on the end is to help with identifying these conditional answers